bzt/modules/jmeter.py

```diff
@@ -1269,6 +1281,19 @@ def __add_assertions(self, children, req):
             children.append(component)
             children.append(etree.Element("hashTree"))
 
+        xpath_assertions = req.config.get("assert-xpath", [])
+        for idx, assertion in enumerate(xpath_assertions):
+            assertion = ensure_is_dict(xpath_assertions, idx, "xpath")
+
+            component = JMX._get_xpath_assertion(assertion['xpath'],
+                                                 assertion.get('validate-xml', False),
+                                                 assertion.get('ignore-whitespace', True),
```

Comment:
If assert ignores whitespaces, why extractor does not? Makes sense to use same default logic.